.github/workflows/greetings.yml

```diff
@@ -6,8 +6,8 @@ jobs:
   greeting:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/first-interaction@v1
-      with:
-        repo-token: ${{ secrets.GITHUB_TOKEN }}
-        issue-message: 'Thank you for reporting your first issue. My master will take a look at his earliest convenience. - GitHub Bot'
-        pr-message: 'Thank you for submitting your first PR. My master will take a look at his earliest convenience. - GitHub Bot'
+      - uses: actions/first-interaction@v1
```

Comment:
is this change as a result of the new pre-commit hook?

Reply:
The file is the same of your. You can check, it is only formatted.

Reply:
Yes that's what I was asking about. 
I don't see why it needed to be formatted.
I don't want prettier integration (yet)

Reply:
The prettier task automatically apply its spaces and indent rules, according its standard